src/main/java/org/docksidestage/app/web/profile/ProfileBean.java

```diff
@@ -0,0 +1,90 @@
+package org.docksidestage.app.web.profile;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.docksidestage.dbflute.exentity.Member;
+import org.docksidestage.dbflute.exentity.MemberService;
+import org.docksidestage.dbflute.exentity.Purchase;
+
+/**
+ * @author y.shimizu
+ */
+public class ProfileBean {
+
+    private String memberName;
```

Comment:
suggestion: Can the fields be final?
Because they are initialized at constructor and nobody may call their setters.